compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -515,7 +437,10 @@ TR::Register *OMR::X86::TreeEvaluator::fpReturnEvaluator(TR::Node *node, TR::Cod
 
    const TR::X86LinkageProperties &linkageProperties = cg->getProperties();
    TR::RealRegister::RegNum machineReturnRegister =
-      (returnRegister->isSinglePrecision())? linkageProperties.getFloatReturnRegister() : linkageProperties.getDoubleReturnRegister();
+         (returnRegister->isSinglePrecision())? linkageProperties.getFloatReturnRegister() : linkageProperties.getDoubleReturnRegister();
+
+   TR_ReturnInfo returnInfo = (returnRegister->isSinglePrecision()) ? TR_FloatXMMReturn : TR_DoubleXMMReturn;
```

Comment:
Can you sink this down to where it is actually used?  It makes the code more readable.